cidc_api/emails.py

```diff
@@ -86,13 +98,21 @@ def new_upload_alert(upload: Union[AssayUploads, ManifestUploads]) -> dict:
         <li><strong>trial id:</strong> {upload.trial_id}</li>
         <li><strong>type:</strong> {upload_type}</li>
         <li><strong>uploader:</strong> {upload.uploader_email}</li>
-    </ul
+    </ul>
     """
 
     email = {
         "to_emails": [CIDC_MAILING_LIST],
         "subject": subject,
         "html_content": html_content,
+        "attachments": [
+            {
+                "content": base64.b64encode(conf.encode()).decode(),
```

Comment:
This line is very hard to parse - can you add a comment describing what is going on here?